### PR TITLE
Fix Zcash typo

### LINKED ISSUE #2298: ZCash should be Zcash
Small style issue but I noticed in DCRDEX Zcash is written as ZCash (camel case) almost everywhere and I've seen people in the community (Decred Magazine) writing it that way too lately but apparently we're the only ones doing that.

Some references:
https://z.cash/
https://github.com/zcash/zcash
https://coinmarketcap.com/currencies/zcash/
https://en.wikipedia.org/wiki/Zcash

### PR DESCRIPTION
**Pull request by ukane-philemon**
Closes #2298

### CODE REVIEW

File: docs/wiki/Client-Installation-and-Configuration.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ DEX client user interface as described in the next section.
 Depending on which assets you wish to use, you have different choices for wallet
 software. There are native/built-in light wallet options for Bitcoin and Decred,
 an external light wallet option for Litecoin, and full-node support for all
-other assets including: Bitcoin, Decred, Litecoin, ZCash, Dogecoin, Bitcoin
+other assets including: Bitcoin, Decred, Litecoin, Zcash, Dogecoin, Bitcoin
```
Comment (chappjc) [inline docs/wiki/Client-Installation-and-Configuration.md:80]: Please put the wiki edit in a separate commit.  I have to keep a sub-tree up-to-date and it's easiest with wiki commits isolated.

File: docs/wiki/Client-Installation-and-Configuration.md [hunk 2]
```diff
@@ -95,7 +95,7 @@ wallet.
 3. **Litecoin.** Either [Litecoin Core v0.21.x](https://litecoin.org/) or
    [Electrum-LTC v4.2.x](https://electrum-ltc.org/) are supported.
 4. **Dogecoin.** [Dogecoin Core v1.14.5+](https://dogecoin.com/).
-5. **ZCash.** [zcashd v5.1](https://z.cash/download/).
+5. **Zcash.** [zcashd v5.1](https://z.cash/download/).
```
Comment (chappjc) [inline docs/wiki/Client-Installation-and-Configuration.md:98]: While here, please change the zcashd version to v5.4.2.  That is the minimum as of d1c45817f3d22db3d93a3dbc9cfb7715546ec2c1

**Review by chappjc — CHANGES_REQUESTED** [review]
Thanks.  Minor edit and split the wiki commit please.

File: docs/wiki/Client-Installation-and-Configuration.md (hunk 1 above)
Comment (ukane-philemon) [reply docs/wiki/Client-Installation-and-Configuration.md:80]: okay.

**Review by chappjc — APPROVED** [review]
